src/Views/Dog/Dog.js

```diff
@@ -2,13 +2,16 @@ import React, { useState, useEffect } from 'react';
 import Header from '../../Components/Header/Header';
 import DogDetail from '../../Components/DogDetail/DogDetail';
 import { getDog } from '../../services/DogData';
-import { useParams } from 'react-router-dom';
+import { Link, useParams, useHistory } from 'react-router-dom';
 import './Dog.css';
+import { deleteDog } from '../../services/DogData';
 
 export default function Dog() {
   const [dog, setDog] = useState([]);
```

Comment:
dog is an object, it would make sense to set the initial state to an empty object `{}`